### ISSUE TITLE
Part of game trickier for colorblind people than it should be.

### ISSUE THREAD

**Issue by framingnoone** [anegostudios/VintageStory-Issues#837]
**Game Version:** 1.4.8
**Platform:** Linux
**Modded:** No

### Description
Slightly difficult to see difference between green and orange(red?) voxels when making clay things
### How to reproduce
1. Attempt to make a clay pot of some sort
2. Green voxels showing where to put clay appear to be almost the same color as orange voxels showing where there is clay that shouldn't be there.

### Expected behavior
Would be nice if there was a way to change the colors to something clearer for somewhat colorblind people to see. Maybe a blue instead of a orange(red?). Managed to make the pot eventually, but this may be more of an issue for other people.

### Screenshots
Attach a screenshot if it helps explain the issue / shows a visual bug.

### Logs
No crash

**Comment by copygirl** [comment]
There are a lot of different types of color blindness. It would be useful to have some developer settings to simulate color blindness to spot / understand these issues.

Perhaps the solution is for players to change the color of the highlights in voxel crafting recipes. Or maybe there is a completely different way to display which voxels need to be added or removed that does not rely out colored outlines - which can even be problematic with people who have "normal" vision.

**Comment by chriswa** [comment]
In case it helps anyone, here's a [web application](http://www.color-blindness.com/coblis-color-blindness-simulator/) which allows you to upload images (e.g. screenshots) to simulate what people with different types of color blindness see.
According to [this source](https://www.color-blindness.com/2006/04/28/colorblind-population/), 99.8% of people who experience some kind of color blindness (who make up 4.25% of all people) either have trouble seeing red or they have trouble seeing green. (The term "red-green colorblind" is a bit confusing.)
It would be nice to have a Graphics Setting which changes the voxel outline colors between Red-Blind/Protanopia -friendly, and Green-Blind/Deuteranopia -friendly colors. More options may be nice, but these two would help out the vast majority of people.

**Comment by copygirl** [comment]
A solution to the color issue is to have a completely different shape to distinguish voxels that need to be removed (use a cross instead) versus ones that need to be added (keep it as a cube?).

**Comment by ApacheTech** [comment]
@framingnoone, I have made a mod, (Accessibility Tweaks)[https://mods.vintagestory.at/show/mod/665]; which allows you to tweak the colours for the game. It includes profiles for all 9 different forms of colour vision ability, using the Dalton formula. I've had some good feedback from people using is so far. I hope it can work for you.

**Comment by Craluminum2413** [comment]
There are 3 profiles for wireframe color since 1.19
